package.json

```diff
@@ -29,26 +29,31 @@
 		"@babel/core": "^7.17.10",
 		"@babel/preset-env": "^7.17.10",
 		"@babel/preset-react": "^7.16.7",
+		"@testing-library/jest-dom": "^5.16.4",
+		"@testing-library/react": "^13.2.0",
 		"babel-loader": "^8.2.5",
 		"eslint": "^8.15.0",
 		"eslint-config-airbnb": "^19.0.4",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-import": "^2.26.0",
+		"eslint-plugin-jest": "^26.2.2",
 		"eslint-plugin-jsx-a11y": "^6.5.1",
 		"eslint-plugin-react": "^7.29.4",
 		"eslint-plugin-react-hooks": "^4.5.0",
 		"html-webpack-plugin": "^5.5.0",
 		"husky": "^7.0.4",
 		"jest": "^28.1.0",
+		"jest-environment-jsdom": "^28.1.0",
 		"lint-staged": "^12.4.1",
 		"prettier": "2.6.2",
 		"webpack": "^5.72.0",
 		"webpack-cli": "^4.9.2",
 		"webpack-dev-server": "^4.9.0"
 	},
 	"dependencies": {
-		"enzyme": "^3.11.0",
-		"enzyme-adapter-react-16": "^1.1.1",
+		"@testing-library/jest-dom": "^5.16.4",
+		"cors": "^2.8.5",
```

Comment:
you shouldn't have `@testing-library/jest-dom` in the dependencies as it's a dev dependency and it's already there. But no big deal